### ISSUE TITLE
Genre Edits in Get Info Not Saving

### ISSUE THREAD

**Issue by animeditor** [openaudible/openaudible#1455]
I've been trying to change Genre using the Get Info tool, and after changing it to Audiobook, it shows in the column but doesn't save, and reverts to what it was before. I've renamed other fields (such as Title) which take, but not Genre. Is this a bug or am I doing something wrong? I'm running OpenAudible 4.4.5 on my Mac Studio running OS 14.6.1

Also is there a future upgrade where we can edit multiple books at the same time or automating it somehow?

**Comment by openaudible** [comment]
I wrote in an email back to you.. but this may not be something we can fix with the current version. When we get a full genre from audible, we use that for the tags, replacing the generic "audiobook". I don't know which iPods this affects, but you're the first to bring it up. Using a bulk mp3 tag editor might help. You can also try m4b vs mp3.. and m4b should automatically get categorized as an audiobook.

**Comment by animeditor** [comment]
Thanks for that email - to close this I went into the id3 tags and found that OpenAudible displays the book_genre rather than genre tag, and explains why, no matter how many times I changed the genre tag, it was only a temporary fix. The problem was in-between the chair and the desk rather than the desk and the wall in this case. The iPod that I'm syncing all my audiobooks with displays the genre tag, so I ended up getting what I wanted after all. You are also correct in saying that the m4b classifies it as an audiobook and the iPod reports as such. Thanks!
